### PR TITLE
add bootnodes and static nodes content

### LINKED ISSUE #151: Add bootnodes and static nodes content
Similar to Besu, we should add information about bootnodes and static nodes to the guide.

Information is currently not easy to find

### PR DESCRIPTION
**Pull request by alexandratran**
## Pull Request Description

Add pages to define bootnodes and static nodes and how to configure them, similar to Besu docs.

## Fixed Issue(s)

fixes #151

### CODE REVIEW

**Review by antonydenyer — APPROVED** [review]
LGTM

File: docs/HowTo/Configure/StaticNodes.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+1. Start GoQuorum with discovery disabled using [`--nodiscover`](https://geth.ethereum.org/docs/interface/command-line-options).
+
+To update the list of static peers at runtime, use the
+[`admin_addPeer`](https://etclabscore.github.io/core-geth/JSON-RPC-API/modules/admin/#admin_addpeer) and
+[`admin_removePeer`](https://etclabscore.github.io/core-geth/JSON-RPC-API/modules/admin/#admin_removepeer) JSON-RPC API methods.
+
+!!! note
+
+    Runtime modifications of static nodes are not persisted between runs.
```
Comment (rolandtyler) [inline docs/HowTo/Configure/StaticNodes.md:45]:
```suggestion
    Runtime modifications of static nodes do not persist between runs.
```

File: docs/HowTo/Configure/StaticNodes.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+## Configure static nodes
+
+To configure a network of static nodes:
+
+1. List the [enode URLs](https://eth.wiki/en/fundamentals/enode-url-format) of the nodes in the
+   [`static-nodes.json` file](#static-nodesjson-file).
+
+    !!! note
+
+        When to specifying enodes in `static-nodes.json`, you can use either DNS names or IP addresses.
```
Comment (rolandtyler) [inline docs/HowTo/Configure/StaticNodes.md:31]:
```suggestion
        When specifying enodes in `static-nodes.json`, you can use either DNS names or IP addresses.
```

File: docs/HowTo/Configure/Bootnodes.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+    ```bash
+    --bootnodes=enode://<publicKeyBootnode1>@<ipBootnode1>:30303,<publicKeyBootnode2>@<ipBootnode2>:30303
+    ```
+
+!!! tip
+
+    Having each bootnode list the other bootnodes increases the speed of discovery.
+    Nodes ignore their own enode in the bootnodes list so it isn't required to specify different bootnode lists to the
+    bootnodes themselves.
```
Comment (rolandtyler) [inline docs/HowTo/Configure/Bootnodes.md:75]:
```suggestion
### Specifying multiple bootnodes

To allow for continuity in the event of failure, specify all bootnodes in a comma-separated list using the
[`--bootnodes`](https://geth.ethereum.org/docs/interface/command-line-options) option for each node, including the bootnodes. 

!!! example

    If your network has two bootnodes, pass the following parameter to all nodes, including the bootnodes.

    ```bash
    --bootnodes=enode://<publicKeyBootnode1>@<ipBootnode1>:30303,<publicKeyBootnode2>@<ipBootnode2>:30303
    ```

!!! tip

    Having each bootnode include the other bootnodes increases the speed of discovery.
    Nodes ignore their own enode in the bootnodes list so you can use the same bootnode list for all bootnodes.
```

File: docs/HowTo/Configure/Bootnodes.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+### Adding and removing bootnodes
```
Comment (rolandtyler) [inline docs/HowTo/Configure/Bootnodes.md:77]: The title says removing bootnodes, but it doesn't actually cover it.  Is it as simple as taking them off the list or are there any other concerns?

**Review by rolandtyler — COMMENTED** [review]
A few comments, let me know if you have any questions.  Thanks.

**Review by rolandtyler — APPROVED** [review]
LGTM

**Pull request merged**